templates/legal/websites.html

```diff
@@ -116,7 +116,7 @@ <h1>Canonical websites</h1>
           <a href="https://snapcraft.io">snapcraft.io</a>
         </li>
         <li class="p-list__item">
-          <a href="https://tutorials.ubuntu.com">tutorials.ubuntu.com</a>
+          <a href="https://www.ubuntu.com/tutorials">www.ubuntu.com/tutorials</a>
```

Comment:
```suggestion
          <a href="/tutorials">www.ubuntu.com/tutorials</a>
```
we don't need the domain on the actual href